src/me/StevenLawson/TotalFreedomMod/Commands/Command_opme.java

```diff
@@ -1,20 +1,26 @@
 package me.StevenLawson.TotalFreedomMod.Commands;
 
+import me.StevenLawson.TotalFreedomMod.TFM_AdminList;
 import me.StevenLawson.TotalFreedomMod.TFM_Util;
+import org.bukkit.ChatColor;
 import org.bukkit.command.Command;
 import org.bukkit.command.CommandSender;
 import org.bukkit.entity.Player;
 
-@CommandPermissions(level = AdminLevel.SUPER, source = SourceType.ONLY_IN_GAME)
-@CommandParameters(description = "Automatically ops user.", usage = "/<command>")
+@CommandPermissions(level = AdminLevel.ALL, source = SourceType.ONLY_IN_GAME)
+@CommandParameters(description = "Automatically ops the sender.", usage = "/<command>")
 public class Command_opme extends TFM_Command
 {
     @Override
     public boolean run(CommandSender sender, Player sender_p, Command cmd, String commandLabel, String[] args, boolean senderIsConsole)
     {
+         if (!TFM_AdminList.isAdminImpostor(sender_p))
```

Comment:
Please look at this, you dont even need to test this to know that there is no way that will compile...